grafana/overlays/moc/smaug/dashboards/landing-page/landing-page.yaml

```diff
@@ -0,0 +1,2312 @@
+apiVersion: integreatly.org/v1alpha1
```

Comment:
could you please add the json for this dashboard as a configmap instead? you can keep the configmap in the [moc/smaug](https://github.com/operate-first/apps/tree/master/grafana/overlays/moc/smaug) overlay.

Reply:
you will need to reference that configmap in the grafana spec for [moc/smaug](https://github.com/operate-first/apps/blob/master/grafana/overlays/moc/smaug/grafana-oauth.yaml#L8) as:
```yaml
configMaps:
    - MY_CONFIGMAP_NAME
config:
    dashboards:
      default_home_dashboard_path: "/etc/grafana-configmaps/MY_CONFIGMAP_NAME/MY_JSON_NAME.json"
```